Automattic/Gravatar-SDK-iOS
# [DO NOT SQUASH] Release 2.0.2 Back Merge (take 2)

# DO NOT SQUASH

### Description

This PR duplicates the work in #471.  The original PR appears to have been squash/merged, which doesn't preserve the commits.  Since we need the commits, make sure this PR is merged via a merge commit, not squashed.

### Testing Steps

- [ ] CI should be 🟢 

Confirm that this branch has all of the expected tags:
```shell
$ git checkout release/2.0.2-additions-back-merge-2
$ git fetch --tags
$ git tag --merged
0.1.0
1.0.0
1.0.1
1.0.1-alpha
2.0.0
2.0.0-rc.1
2.0.1
2.0.2
2.1.0
2.1.1
```

After merging this PR, do the same check on `trunk`.
Before:
```diff
0.1.0
1.0.0
1.0.1
1.0.1-alpha
2.0.0
2.0.0-rc.1
```

After:
```
0.1.0
1.0.0
1.0.1
1.0.1-alpha
2.0.0
2.0.0-rc.1
2.0.1
2.0.2
2.1.0
2.1.1
```
